manuals/1.0/en/BindingAttributes.md

```
@@ -139,6 +139,6 @@ Since annotations cannot be applied to arguments, the first argument of a custom
  */
 public function setCreditCardProcessor(
 	 CreditCardProcessorInterface $processor
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)